src/openrct2/actions/LoadOrQuitAction.cpp

```diff
@@ -18,6 +18,12 @@ LoadOrQuitAction::LoadOrQuitAction(LoadOrQuitModes mode, PromptMode savePromptMo
 {
 }
 
+void LoadOrQuitAction::AcceptParameters(GameActionParameterVisitor& visitor)
```

Comment:
debatable if this is safe to perform from a plugin as it will unload the plugin. This may cause unexpected behaviour.